dashboard/lib/queries/script_activity.rb

```diff
@@ -0,0 +1,31 @@
+class Queries::ScriptActivity
```

Comment:
It'd be nice for methods in this file to have doc comments for return types, since they're different. (Although the names do a pretty good job of this.)